.gitmodules

```diff
@@ -40,3 +40,6 @@
 	branch = release
 	ignore = dirty
 	shallow = true
+[submodule ".\\Externals\\rapidjson"]
+	path = .\\Externals\\rapidjson
```

Comment:
Naming and patching can just use forward slashes like all the other submodules

Reply:
Also add the `branch`, `ignore` and `shallow` properties please

Reply:
Sorry that was just what was automatic with git. Will FIx